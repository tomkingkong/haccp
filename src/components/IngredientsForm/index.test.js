import React from 'react';
import { shallow } from 'enzyme';

import { IngredientsForm } from '.';

describe('IngredientsForm Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<IngredientsForm ingredients={[]}/>);
    expect(wrapper).toMatchSnapshot();
  });
});