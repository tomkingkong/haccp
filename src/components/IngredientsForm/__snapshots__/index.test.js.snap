// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IngredientsForm Component should match snapshot 1`] = `
<div>
  <section
    className="ingredients"
  />
  <form
    className="form"
  >
    <TextField
      id="standard-dense"
      label="Ingredient name"
      name="ingredientName"
      onChange={[Function]}
      required={true}
      select={false}
      value=""
      variant="standard"
    />
    <WithStyles(Button)
      onClick={[Function]}
      variant="contained"
    >
      <pure(AddIcon)
        style={
          Object {
            "color": "forestgreen",
            "marginRight": "0.2rem",
          }
        }
      />
      Add Ingredient
    </WithStyles(Button)>
  </form>
</div>
`;
